pkg/cmd/list/list.go

```diff
@@ -0,0 +1,95 @@
+// Lists the latests builds for a project.
+package list
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bufbuild/connect-go"
+	"github.com/depot/cli/pkg/api"
+	"github.com/depot/cli/pkg/config"
+	"github.com/depot/cli/pkg/project"
+	cliv1 "github.com/depot/cli/pkg/proto/depot/cli/v1"
+	"github.com/docker/cli/cli"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdLast() *cobra.Command {
+	var projectID string
+	var token string
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"l"},
+		Short:   "List builds for a project",
+		Args:    cli.ExactArgs(1),
```

Comment:
I'm not exactly sure how someone like Salto will use this, but it may be nice to provide a `--output` arg where they could choose JSON, CSV, or whatever we want for the default in the CLI.

Reply:
Ok, CSV and JSON output format is now supported via --format